frontend/src/components/header/Header.tsx

```diff
@@ -2,23 +2,28 @@ import { Flex } from "@chakra-ui/react";
 import Logo from "./Logo";
 import { FC } from "react";
 import { AuthButtons } from "../authentication/SignInOrOut";
-import { NavBar } from "./NavBar";
+import NavBar from "./NavBar";
 
 interface HeaderProps {}
 
 export const Header: FC<HeaderProps> = () => {
-	return (
-		<Flex
-			justifyContent="space-between"
-			alignItems="center"
-			h="60px"
-			outline={1}
-		>
-			<Logo />
+  return (
+    <Flex
+      justifyContent="space-between"
+      alignItems="center"
+      h="60px"
+      outline={1}
+      position={"absolute"}
+      top={0}
+      left={0}
+      w={"100dvw"}
+      px={8}
+    >
```

Comment:
Getting a bit chunky, lets export it into a local `styles.ts` folder

Reply:
I'm trying to implement this change now, but a big problem I'm running into is that by styling the component with the style property overrides all the chakra stylings that I'm NOT styling:

When I use style.ts:
![image](https://github.com/jtabba/CDW-1/assets/12403004/676dcc45-78f6-4218-aec4-bfd7cdc806f4)

When I style the component using the chakra props:

![image](https://github.com/jtabba/CDW-1/assets/12403004/8b77d741-8388-4c44-8b49-e88043343f64)